lntest/itest/lnd_channel_backup_test.go

```diff
@@ -1048,6 +1040,13 @@ func testChanRestoreScenario(t *harnessTest, net *lntest.NetworkHarness,
 		t.Fatalf("unable to restore node: %v", err)
 	}
 
+	// Wait for the node to finish starting.
+	ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
+	defer cancel()
+
+	err = dave.WaitForBlockchainSync(ctxt)
```

Comment:
Can be done _within_ the `restroedNodeFunc` so the `Restart()` method itself? May also help tighten up other issues we're seeing in the itests as well.

Reply:
This commit was removed as we now require the node to be started when calling `HarnessNode.start()`.